src/content/get-started/overview.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Installation Overview
+description: How to install Okteto in your own Kubernetes cluster
+sidebar_label: Overview
+id: overview
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Okteto is a platform that simplifies the process of deploying development environments in Kubernetes without requiring the expertise to do this yourself.
+This enables developers to automatically spin up fully-managed development environments that emulate production as closely as possible.
+Okteto eliminates the friction of local development environments, the many deviations that can exist for the same engineering organization, and the troubleshooting that comes with them.
+
+:::info
+If you are interested in using Okteto's SaaS offering, you can ignore this page and skip to [Install the Okteto CLI](get-started/install-okteto-cli.mdx).
+:::
+
+### Getting your Okteto License
+
+A license is mandatory to use Okteto. You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
+
+If you are interested in evaluating Okteto, [sign up for our free 30 days trial](https://www.okteto.com/free-trial/). No credit card required.
+
+## Deploying Okteto
+
+Okteto is distributed as a Helm chart. These cloud-specific guides walk you through deploying the Okteto Helm chart with default values and basic configuration:
+
+- [Install Okteto in Amazon Elastic Kubernetes Service](get-started/cloud-provider-guides/eks.mdx)
+- [Install Okteto in Azure Kubernetes Service](get-started/cloud-provider-guides/aks.mdx)
+- [Install Okteto in Civo](get-started/cloud-provider-guides/civo.mdx)
+- [Install Okteto in Google Kubernetes Engine](get-started/cloud-provider-guides/gke.mdx)
+- [Install Okteto in DigitalOcean Kubernetes](get-started/cloud-provider-guides/digitalocean.mdx)
+
+## Video Tutorials
+
+If you need help installing Okteto on your Kubernetes cluster, we have a playlist of videos guiding you through each of the installation steps. Check them out here:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLVcXGaCKg-Y0XGujg_G7vCIGd_JEAQ4Sq" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Troubleshooting
+
+We've done our best to make Okteto easy to install and operate. [The troubleshooting page](self-hosted/manage/troubleshooting.mdx) has tips and tricks to help troubleshoot common issues.
+
+You can also [reach out to support](https://okteto.com/support) if you need assistance.
+
+## Uninstalling Okteto
```

Comment:
Since we don't show them how to install it on this page, I think we should remove this from here. maybe we can add the uninstall instructions on the troubleshooting section? (or maybe a full section on how to uninstall Okteto?)